main.py

```diff
@@ -43,8 +43,8 @@
 
 app = Flask(__name__, static_url_path='/static')
 
-SENSORS = ["battery_current", "battery_temperature", "battery_voltage", "bms_fault", "gps_course", "gps_dt", "gps_lat",
-           "gps_lon", "gps_speed", "motor_speed", "solar_current", "solar_voltage"]
+# SENSORS = ["battery_current", "battery_temperature", "battery_voltage", "bms_fault", "gps_course", "gps_dt",
```

Comment:
We can probably go ahead and remove this completely, especially since we still also have database_format.json